src/components/applicantProfile/ApplicantProfile.scss

```diff
@@ -7,7 +7,10 @@
     display: flex;
     width: 50%;
     justify-content: space-between;
-    border: 1px solid rgb(146, 146, 146);
+    border: 1px solid #e0e0e0;
+    border-right: 0;
+    border-radius: 4px 0 0 4px;
+    box-shadow: 0 4px 14px rgba(0, 0, 0, 0.055);
     padding: 2em;
```

Comment:
padding in `em` looks very tricky, could it be changed?

Reply:
replaced with rem.